src/low_pass_filter.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file low_pass_filter.cpp
+ * @author Lucien C. Peach (peach@seas.upenn.edu)
+ * @brief Implementation of a discrete-time low-pass filter class.
+ * @date 9/10/2022
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#include "kodlab_mjbots_sdk/low_pass_filter.h"
+#include <math.h>
+#include <Eigen/Geometry>
+
+namespace kodlab
+{
+
+LowPassFilter::LowPassFilter(float dt, float k_frequency_cutoff, Eigen::Vector3f raw_data) {
+
+    dt_ = dt;
+    raw_data_ = raw_data;
+    filtered_data_ = raw_data;
+    k_frequency_cutoff_ = k_frequency_cutoff; 
+
+};
```

Comment:
**[General]** Per [this ISO C++ Standard item](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c49-prefer-initialization-to-assignment-in-constructors), you should use an [initializer list](https://en.cppreference.com/w/cpp/language/constructor) here instead of assignment.